src/modules/assertions/require_created_at.ts

```diff
@@ -0,0 +1,16 @@
+import { context, getOctokit } from "@actions/github";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+	
+export const getCreatedAt = () => { 
+  const github = getOctokit(GITHUB_TOKEN).rest;
+  const createRegex = /^[\s\S]*created:  \d\d\d\d-\d\d-\d\d\n?/g;
+  if !(createRegex.test(context.payload?.pull_request?.body)) {
+    console.warn(`Warning: Please ensure to include in the preamble created date: ${context.payload?.pull_request?.created_at}`);
+  }
+}; 
+
```

Comment:
Please put this in a separate PR so it can be evaluated separate from the CC0 stuff.

Reply:
I tried it, every time I tried it ask me to merge to this PR !!

Reply:
You will want to create a different git branch for each pull request.  Right now you have a single branch that contains many changes, which is why they all show up in the same PR.